### PR TITLE
Bump OCP min version to 4.11

### PR DESCRIPTION
**Pull request by ReToCode**
## Changes
- Bump OCP min version to 4.11 (according to release matrix)
- Will run [CI generation](https://github.com/openshift-knative/serverless-operator/actions/workflows/release-generate-ci.yaml) afterwards

/assign @pierDipi

### CODE REVIEW

File: olm-catalog/serverless-operator/project.yaml [hunk 1]
```diff
@@ -28,9 +28,9 @@ requirements:
     golang: '1.19'
     nodejs: 16.x
     ocpVersion:
-        min: '4.10'
+        min: '4.11'
         max: '4.13'
-        label: 'v4.10'
+        label: 'v4.11'
```
Comment (mgencur) [inline olm-catalog/serverless-operator/project.yaml:33]: Perhaps, also `requirements.kube.minVersion` should be increased to 1.24.0?

Comment (ReToCode) [reply olm-catalog/serverless-operator/project.yaml:33]: 👍

**Pull request merged**